features/manage-position/Create.tsx

```diff
@@ -73,6 +78,19 @@ const Create = () => {
   } = empState;
   const liquidationPriceWarningThreshold = 0.1;
 
+  useEffect(() => {
+    if (gcr !== null) {
+      setBackingCollateralToMin(gcr, Number(tokens));
+    }
+  }, [gcr, tokens]);
+
+  const setBackingCollateralToMin = (_gcr: number, _tokens: number) => {
+    // By default set amount of collateral to the minimum required by the GCR constraint. This
+    // default is intended to encourage users to maximize their capital efficiency.
+    const minBackingCollateral = _gcr * _tokens;
+    setCollateral(minBackingCollateral.toFixed(4));
```

Comment:
nit: `toFixed()` rounds, right? So if it rounds down, will the pre-filled collateral not meet the GCR requirement?

Reply:
yeah good point, but there is no clean solution to "round up": https://helloacm.com/javascripts-tofixed-implementation-without-rounding/

What I can do is naively just add 0.00005 to the number before sending to `.toFixed()`

Reply:
Actually, the cleaner solution might be to NOT round in this function and just round in the JSX render element